packages/client/src/components/domain/shop/ManufacturerListPage.tsx

```diff
@@ -39,5 +41,13 @@ export const ManufacturerListPage = () => {
     },
   ];
 
-  return <Table columns={columns} data={manufacturers} />;
+  return (
+    <Table
+      columns={columns}
+      data={manufacturers}
+      onClick={(item) => {
+        navigate(`/shop/manufacturers/${item.id}/products`);
+      }}
```

Comment:
[nits] 一行のみの場合は中括弧を省略できそうです！

```suggestion
onClick={(item) => navigate(`/shop/manufacturers/${item.id}/products`)}
```